gapic/src/main/com/google/gapid/widgets/Widgets.java

```diff
@@ -453,6 +456,23 @@ public static Text createTextarea(Composite parent, String text) {
     return result;
   }
 
+  public static StyledText createStyledText(Composite parent, String text) {
```

Comment:
Let's give this function a better name, since it creates a very specific `StyledText`. Maybe `createSelectableLabel`?